doc/advanced.dox

```diff
@@ -188,8 +188,8 @@ bbp::rtneuron::RTNeuron::createScene RTNeuron.createScene\endlink.
 
 The environmental variable RTNEURON_LOD_CONFIG_FILE can be set to point to a
 file that contains a LOD specification to use for automatic scenes. The file
-may contain a LOD definition per line using this format: <code>lod_name
-start end</code>. Lines starting with \# are ignored. The valid LOD names are
+may contain a LOD definition per line using this format: \c lod_name
+start end. Lines starting with \# are ignored. The valid LOD names are
```

Comment:
"start end" is no longer inside the code tag?

Reply:
good catch